data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -395,42 +512,80 @@ def _workforce_factor(self) -> bool:
         # AND
         # Where the high school degree achievement rates for adults 25 years and older is less than 95%
         # (necessary to screen out university block groups)
-        workforce_criteria_for_states = (
-            (
-                self.df[
-                    field_names.UNEMPLOYMENT_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.MEDIAN_INCOME_PERCENT_AMI_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                # Note: a high median income as a % of AMI is good, so take 1 minus the threshold to invert it.
-                # and then look for median income lower than that (not greater than).
-                <= 1 - self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.POVERTY_LESS_THAN_100_FPL_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.LINGUISTIC_ISO_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-        )
-        workforce_combined_criteria_for_states = (
+
+        high_scool_achievement_rate_threshold = (
             self.df[field_names.HIGH_SCHOOL_ED_FIELD]
             >= self.LACK_OF_HIGH_SCHOOL_MINIMUM_THRESHOLD
-        ) & workforce_criteria_for_states
+        )
+
+        unemployment_threshold = (
+            self.df[
+                field_names.UNEMPLOYMENT_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
+        )
+
+        median_income_threhsold = (
```

Comment:
nit: median_income_threhsold -> median_income_threshold

Reply:
Thanks for catching that